src/Forms/HTMLEditor/TinyMCEConfig.php

```diff
@@ -209,6 +209,31 @@ class TinyMCEConfig extends HTMLEditorConfig
      */
     protected $contentCSS = null;
 
+
+    /**
+     * List of image size preset that will appear when you select an image. Each preset can have the following:
+     * * `name` to store an internal name for the preset (required)
+     * * `i18n` to store a translation key (e.g.: `SilverStripe\Forms\HTMLEditor\TinyMCEConfig.BESTFIT`)
+     * * `text` that will appear in the button (should be the default English translation)
+     * * `width` which will define the horizontal size of the preset. If not provided, the preset will match the
+     *   original size of the image.
+     * @var array
+     * @config
+     */
+    private static $image_size_presets = [
+        [
```

Comment:
It doesn't hurt to define this here ... it doesn't do anything if you don't have asset-admin installed.

Although one could argue that it would make more sense to define those values in asset-admin.

Reply:
Could we move it to YAML though?

Reply:
Done.